geoportal/c2cgeoportal_geoportal/views/login.py

```diff
@@ -81,6 +81,9 @@ def __init__(self, request: pyramid.request.Request):
 
     def _functionality(self) -> dict[str, list[str | int | float | bool | list[Any] | dict[str, Any]]]:
         functionality = {}
+        if hasattr(self.request, "user_") and self.request.user_ is not None and models.DBSession is not None:
+            models.DBSession.add(self.request.user_)
```

Comment:
I would think this should be done in get_user_from_request, wreid doing this in _functionality